include/powsybl/iidm/ThreeWindingsTransformerLeg.hpp

```diff
@@ -110,17 +134,15 @@ class Leg : public virtual RatioTapChangerHolder, public virtual PhaseTapChanger
 private:
     const std::string& getTypeDescription() const;
 
-    void setCurrentLimits(std::nullptr_t side, std::unique_ptr<CurrentLimits> limits);
-
     Leg& setTransformer(ThreeWindingsTransformer& transformer);
 
-    friend class CurrentLimitsAdder<const std::nullptr_t, Leg>;
-
     friend class iidm::ThreeWindingsTransformer;
 
 private:
     stdcxx::Reference<ThreeWindingsTransformer> m_transformer;
 
+    std::unique_ptr<OperationalLimitsHolder> m_operationalLimitsHolder;
```

Comment:
This is not possible because the holder cannot be instanciated at the same time than leg, because the identifiable is not known while building the leg. It is set later, when calling `Leg::setTransformer` in Java:
```
        void setTransformer(ThreeWindingsTransformerImpl transformer) {
            this.transformer = transformer;
            operationalLimitsHolder = new OperationalLimitsHolderImpl(transformer, "limits" + legNumber);
        }
```

Reply:
Updated by using a `stdcxx::Reference<Identifiable>` which is set later in `setTransformer`